Sources/Commands/SwiftTool.swift

```diff
@@ -419,6 +424,11 @@ public class SwiftTool<Options: ToolOptions> {
             )
         })
     }()
+
+    enum ExecutionStatus {
+        case success
+        case failure(Int)
```

Comment:
We only have one exit code currently, right? If so, lets remove the (Int) for now.